packages/fxa-settings/src/models/integrations/factory.ts

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useContext } from 'react';
+import { AppContext } from '../AppContext';
+import { IntegrationFactory } from '../../lib/integrations/integration-factory';
+import { DefaultIntegrationFlags } from '../../lib/integrations/integration-factory-flags';
+
+export function CreateIntegrationFlags() {
+  const { urlQueryData } = useContext(AppContext);
+
+  if (!urlQueryData) {
+    throw new Error('Are you forgetting an AppContext.Provider?');
+  }
+
+  return new DefaultIntegrationFlags(urlQueryData);
+}
+
+export function CreateIntegrationFactory() {
```

Comment:
Same here. This is `export`ed but is currently only used below by this file on L28.